src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -862,6 +862,45 @@ async def SendCommand(self, nodeid: int, endpoint: int, payload: ClusterObjects.
             interactionTimeoutMs=interactionTimeoutMs, busyWaitMs=busyWaitMs, suppressResponse=suppressResponse).raise_on_error()
         return await future
 
+    async def SendBatchCommands(self, nodeid: int, commands: typing.List[typing.Tuple[int, ClusterObjects.ClusterCommand, typing.Type]],
+                                timedRequestTimeoutMs: typing.Union[None, int] = None,
+                                interactionTimeoutMs: typing.Union[None, int] = None, busyWaitMs: typing.Union[None, int] = None,
+                                suppressResponse: typing.Union[None, bool] = None):
+        '''
+        Send a batch of cluster-object encapsulated commands to a node and get returned a future that can be awaited upon to receive
+        the responses. If a valid responseType is passed in, that will be used to deserialize the object. If not,
+        the type will be automatically deduced from the metadata received over the wire.
+
+        nodeId: Target's Node ID
+        commands: A list of tuples of type (endpoint, cluster-object, response-type):
```

Comment:
Why is response-type needed? In `SendCommand` we have:

> If a valid responseType is passed in, that will be used to deserialize the object. If not,
        the type will be automatically deduced from the metadata received over the wire

Suggest:

- Making a DataClass for a commands (rather than needed explicit tuples which are hard to read)
- Make the response-type None by default by default